appengine/mongo/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "appengine-mongo",
+  "description": "An example of using MongoDB with Node.js on Google App Engine.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "engines": {
+    "node": "~0.12.7"
```

Comment:
As of wed, 4.2.1 is now the default version on the image.  Next wed it will be 4.2.2.  Maybe more of a @jmdobry question, but we should do a pass to bring this all up to 4.x.